contrib/charts/navigator/templates/rbac.yaml

```diff
@@ -101,7 +101,7 @@ items:
     resourceNames: ["navigator-controller"]
   - apiGroups: [""]
     resources: ["events"]
-    verbs:     ["create", "update"]
+    verbs:     ["create", "update", "patch"]
```

Comment:
❓ I wonder if we need the "update" verb in fact?

Looks like service catalog has the same rules, so fine:
 * https://github.com/kubernetes-incubator/service-catalog/blob/master/charts/catalog/templates/rbac.yaml#L79